app/server.go

```diff
@@ -138,8 +140,8 @@ func NewServer(options ...Option) (*Server, error) {
 		goroutineExitSignal:     make(chan struct{}, 1),
 		RootRouter:              rootRouter,
 		licenseListeners:        map[string]func(){},
-		sessionCache:            utils.NewLru(model.SESSION_CACHE_SIZE),
-		seenPendingPostIdsCache: utils.NewLru(PENDING_POST_IDS_CACHE_SIZE),
+		sessionCache:            lru.NewLru(model.SESSION_CACHE_SIZE),
```

Comment:
Probably we can add the a `CacheProvider` field in the server object and initialize it as soon as we have the configuration (because in the future we will need that to configure the proper cache provider). And initialize this caches using the `CacheProvider`, not the lru directly.

Reply:
Done. I thought that we could call cacheProvider.close() inside the shutdown method of the server. What do you think?